projects/core/src/lib/colors/color-palette-editor/color-palette-editor.component.html

```diff
@@ -0,0 +1,13 @@
+<cdk-virtual-scroll-viewport appendOnly itemSize="80" class="tabs-viewport">
+    <div *cdkVirtualFor="let c of allColors; let i = index" class="grid-container">
```

Comment:
![grafik](https://user-images.githubusercontent.com/4947061/233296656-53a7b637-f8c9-465e-b5f2-6d201392444d.png)

For NDVI, although you know how many items there are in the list, the scrollbar isn't sized correctly. It flickers when you scroll down.

Here (https://material.angular.io/cdk/scrolling/examples), in example number one, it is done correctly.

Reply:
Could this be different on Mac? For me (Mac Chrome + Safari), both in the example you linked and in this PR, the scrollbar is fairly tiny from the start, and doesn't flicker or change it size. Unfortunately I don't have access to a Windows PC right now so I can't test this

Reply:
I don't have a Windows PC too for testing, but I use Firefox, which looks as expected in the example I linked, but wrong (cf. screenshot) in Geo Engine.

Reply:
Fixed in [7fbcc6e](https://github.com/geo-engine/geoengine-ui/pull/614/commits/7fbcc6edc42516577798dfccc44d9fedc99f57fd), the scroll bar has the right length from the start and it's possible to scroll smoothly